modules/imaging_browser/php/NDB_Menu_Filter_imaging_browser.class.inc

```diff
@@ -56,24 +56,37 @@ class NDB_Menu_Filter_Imaging_Browser extends NDB_Menu_Filter
      */
     function _setupVariables()
     {
+        $config = NDB_Config::singleton();
+        $DB = Database::singleton();
+        $scan_types = $config->getSetting('tblScanTypes');
+
+        foreach ($scan_types as $type) {
+            $id = $DB->pselectOne(
+                "SELECT ID FROM mri_scan_type mst
+                     WHERE mst.Scan_type='" . $type . "'",
+                array()
```

Comment:
Why not just have a pselect (not one) to do it all in one query? The `mri_scan_type` table is small enough that if need be since prepared statements with in clauses are difficult you could just select everything and do an array_filter to filter it to the `tblScanTypes`.

`$type` should also be a prepared parameter, not a string concatenation.